lxd/images.go

```diff
@@ -874,23 +874,24 @@ func autoUpdateImages(d *Daemon) {
 	for _, fingerprint := range images {
 		id, info, err := dbImageGet(d.db, fingerprint, false, true)
 		if err != nil {
-			logger.Error("Error loading image", log.Ctx{"err": err, "fp": fingerprint})
+			logger.Error("Error loading image", log.Ctx{"err": err, "fp": info.Fingerprint})
```

Comment:
Hmm, this feels like a very bad idea. There's no guarantee that info.Fingerprint will be set on error.